modules/runner/src/integrationTest/java/org/apache/ignite/internal/sql/engine/datatypes/varbinary/ItVarBinaryIndexTest.java

```diff
@@ -18,20 +18,132 @@
 
 package org.apache.ignite.internal.sql.engine.datatypes.varbinary;
 
+import static org.apache.ignite.internal.sql.engine.util.QueryChecker.containsIndexScan;
+import static org.apache.ignite.lang.IgniteStringFormatter.format;
+
+import java.util.stream.Stream;
 import org.apache.calcite.sql.type.SqlTypeName;
 import org.apache.ignite.internal.sql.engine.datatypes.DataTypeTestSpecs;
 import org.apache.ignite.internal.sql.engine.datatypes.tests.BaseIndexDataTypeTest;
 import org.apache.ignite.internal.sql.engine.datatypes.tests.DataTypeTestSpec;
+import org.apache.ignite.internal.sql.engine.datatypes.tests.TestTypeArguments;
 import org.apache.ignite.internal.sql.engine.util.VarBinary;
+import org.apache.ignite.internal.util.IgniteUtils;
+import org.junit.jupiter.api.Assumptions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Named;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
 
 /**
  * Tests for queries that use indexes with {@link SqlTypeName#VARBINARY} type.
  */
 public class ItVarBinaryIndexTest extends BaseIndexDataTypeTest<VarBinary> {
 
+    @BeforeAll
+    public void createTable() {
+        runSql("CREATE TABLE varbinary_fixed_length(id INTEGER PRIMARY KEY, test_key VARBINARY(10))");
```

Comment:
VARBINARY is variable length BINARY type. 
What does `varbinary_fixed_length` mean?

Reply:
If you mean we could create columns as 
```
col1 VARBINARY,
col2 VARBINARY(10)
```
then, actually, col1 will have a max length as well.
We always create VARBINARY/VARCHAR column with specific length, which is 65k by default as I remember.

Reply:
@AMashenkov 

>VARBINARY is variable length BINARY type.
>What does varbinary_fixed_length mean?

Right now VARBINARY(n) works exactly the same way as does BINARY(n).  If this is a bug, I can replace `VARBINARY(n)` with BINARY(n) here, I could file an issue to fix this.

Reply:
Removed varbinary_fixed_length as it duplicates test cases defined in base class.